web/src/beta/features/ProjectSettings/innerPages/PublicSettings/PublicSettingsDetail.tsx

```diff
@@ -232,29 +233,25 @@ const PublicSettingsDetail: React.FC<Props> = ({
         <TitleWrapper size="body" weight="bold">
           {t("Custom Domain")}
         </TitleWrapper>
-        { isPublished &&
-          extensions &&
-          extensions.length > 0 &&
-          accessToken ? (
-            <ExtensionComponent
-              typename={settingsItem.__typename || ""}
-              {...(settingsItem.__typename === "Project"
-                ? {
-                    projectId: settingsItem.id,
-                    projectAlias: settingsItem.alias
-                  }
-                : {
-                    storyId: settingsItem.id,
-                    storyAlias: settingsItem.alias
-                  })}
-              lang={currentLang}
-              theme={currentTheme}
-              accessToken={accessToken}
-              onNotificationChange={onNotificationChange}
-              version="visualizer"
-            />
-          )
-           : (
+        {isPublished && extensions && extensions.length > 0 && accessToken ? (
```

Comment:
Please update the logic for extensions here.
- Filter extensions with `type === "publication"`
- Corrent the conditions, don't show this section if there's no extension
```
if(extensions && extensions.length > 0 && accessToken){
  if(isPublished) return <ExtensionComponent />
  else return <ContentDescription />
}
```